ReviewChecklist.md

```diff
@@ -7,6 +7,10 @@ and solve common issues before they are pointed out in the review.
 It is also meant to serve as a final checklist for reviewers to go through before approving a PR.
 
 ## Before You Submit a PR
+- [ ] **Do you have any other open PRs?**
+    Work on a PR is not done until it is merged or closed.
+    Our reviewing capacity is limited, so we ask external contributors to work on no more than one PR at a time.
```

Comment:
```suggestion
    Our reviewing capacity is limited, so we require that external contributors work on **no more than one PR at a time**.
```
We can reword this if need be, but I feel that 'ask' is too soft of a word to use in this context, and makes it seem optional rather than mandatory. In addition, we should mention that if an external contributor has open more than one PR concurrently, the oldest (chronologically) will be left open, and the rest will be closed.

Reply:
I didn't want to sound too harsh here (regardless of quality, these are people basically donating their time to us). Also this is more of a rule of thumb than a strict requirement. There will often be times when someone has more than one. #13743 is a good example - I requested it myself, so I'm not going to close it just because the submitter has another PR :) In some cases we may also ask people to extract a part as a separate PR.

But ok, I can make it stronger. Maybe it's ok to say it's strict but not really be 100% strict about it...

Reply:
Maybe add `unless otherwise specified/requested` or some such, but I'd err on the side of being strict so we don't end up bloating our PR queue again :man_shrugging:

Reply:
Added.